src/chunk.c

```diff
@@ -2985,6 +3023,20 @@ ts_chunk_set_schema(Chunk *chunk, const char *newschema)
 	return chunk_update_form(&chunk->fd);
 }
 
+bool
+ts_chunk_add_status(Chunk *chunk, int status)
```

Comment:
```suggestion
ts_chunk_add_status(Chunk *chunk, int32 status)
```
This should match `FormData_chunk`.